tests/test-pads.stanza

```diff
@@ -0,0 +1,59 @@
+#use-added-syntax(jitx)
+defpackage pads :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/land-patterns
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+pcb-landpattern lp (n:Int) :
+  for i in 0 to n by 6 do :
+    val d = to-double(i) * 2.0
```

Comment:
`* 12.0` instead?
Have you tried with an input `n` larger than `6`?

Reply:
No. Why * 12?

Reply:
My bad